docs/sources/operations/caching.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Caching 
+menuTitle: Caching 
+description: Enable and configure memcached for caching. 
+weight: 100
+keywords:
+  - memcached
+  - caching
+---
+<!-- For more information about how to populate front matter, see [Topic front matter]({{< relref "../../front-matter/" >}}). -->
+
+# Enable and configure Memcached
+
+Loki supports caching of index writes and lookups, chunks and query results to
+speed up query performance. This sections describes the recommended Memcached
+configuration to enable caching.
+
+## Before you begin
+
+- It is recommended to deploy four dedicated Memcached clusters.
+- As of 2023-02-01 the `memcached:1.6.17-alpine` is recommended.
+- Consult the Loki ksonnet [memcached](https://github.com/grafana/loki/blob/main/production/ksonnet/loki/memcached.libsonnet) deployment and the ksonnet [memcached library](https://github.com/grafana/jsonnet-libs/tree/master/memcached).
+
+## Steps
+
+To enable and configure Memcached:
+
+1. Deploy each Memcached service with at least three replicas and configure
+   each as follows:
+  1. Chunk cache `--memory-limit=4096 --max-item-size=2m --conn-limit=1024`
+  1. Index lookup cache `--memory-limit=1024 --max-item-size=5m --conn-limit=1024`
+  1. Index write cache `--memory-limit=1024 --max-item-size=1m --conn-limit=1024`
+  1. Query result cache `--memory-limit=1024 --max-item-size=5m --conn-limit=1024`
+
+1. Configure Loki to use the cache.
+  1. If the Helm chart is used
+    1. set `memcached.chunk_cache.host` to the Memecache address of for the chunk cache.
+    1. set `memcached.results_cache.host` to the Memecache address of for the query result cache.
+    1. Ensure that the connection limit of Memcached is at least `number_of_clients? * max_idle_conns`.
+  1. If the Loki configuration is used
```

Comment:
The following defaults are from our ksonnet deployment. Should we use these in Helm, @trevorwhitney?

Reply:
Seems the fifocache should suffice.

Reply:
@jeschkies we already have these same defaults for the chunks and results cache. Currently the helm chart does not set up a `write_dedupe_cache`, so we would have to add that section completely including the defaults.

As for the index query cache we might need to revisit that. For `write` pods there's no need, as it [defaults to an in memory cache](https://github.com/grafana/loki/blob/9920a27f06b9a808bcd0d7323a55041bc343a8bc/pkg/loki/modules.go#L523). However the store is also use by `read` pods, and I'm not sure I understand the significance of the index query cache in query flows. Is in memory sufficient? We may need to add a section for that to the helm chart.